Tutorials/Making_Maps_with_Cartopy.ipynb

```diff
@@ -18,21 +18,14 @@
    "cell_type": "code",
```

Comment:
<p>I would explain why you are calling it the "tripole" region. ACCESS-newbies might not know that we have a tripole there</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>